bups/scheduler/systemd_user.py

```diff
@@ -0,0 +1,82 @@
+import ConfigParser
+import io
+import os
+import tempfile
+
+config_dir = os.path.join(os.path.expanduser("~"), ".config/systemd/user/")
+
+def is_available():
+    return os.path.exists(config_dir)
```

Comment:
Ok, may be I should check if systemctl executable is available. What do you think?